.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Add .history to .gitignore.

Then remove files from git.

https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore